experimental/language-generation/README.md

```diff
@@ -164,8 +170,8 @@ Nightly release packages:
 
     |  Old  | New |
     |-------|-----|
-    | # myTemplate <br/> - I have {user.name} as your name |  # myTemplate <br/> - I have ${user.name} as your name |
-    | # myTemplate <br/> - [ackPhrase] <br/><br/> # ackPhrase <br/> - hi <br/>- hello | # myTemplate <br/> - ${ackPhrase()} <br/><br/> # ackPhrase <br/> - hi <br/>- hello | 
+    | # myTemplate <br/> - I have {user.name} as your name |  # myTemplate <br/> - I have @{user.name} as your name |
```

Comment:
>@ [](start = 90, length = 1)

Don't we need to say that we went from @{ } => ${ } ?

Reply:
This is 4.7 PREVIEW change.